MinecraftClient/ChatBot.cs

```diff
@@ -662,6 +664,45 @@ public bool GetInventoryEnabled()
         {
             return Handler.GetInventoryEnabled();
         }
+        public Dictionary<int, Container> GetInventories()
+        {
+            return Handler.GetInventories();
+        }
+        /// <summary>
+        /// Use item in hand
+        /// </summary>
+        protected void UseItemInHand()
+        {
+            Handler.UseItemOnHand();
+        }
+        /// <summary>
+        /// start Sneaking
+        /// </summary>
+        protected bool Sneak()
+        {
+            return SendAction(Protocol.ActionType.StartSneaking);
+        }
+        /// <summary>
+        /// Sends UnSneak
+        /// </summary>
+        protected bool UnSneak()
+        {
+            return SendAction(Protocol.ActionType.StopSneaking);
+        }
+        /// <summary>
+        /// Send Entity Action
+        /// </summary>
+        private bool SendAction(Protocol.ActionType action)
```

Comment:
I think `SendEntityAction` would be clearer. Since this is the Bot API, it is almost impossible to change function names after creating the methods without breaking user-created bots, so we should choose them wisely. 😄